### PR TITLE
Change COGUTIL_DATA_DIR to DATADIR

### PR DESCRIPTION
**Pull request by kasimebrahim**

### CODE REVIEW

**Comment by ngeiswei** [comment]
I'm not confident enough to merge, @vsbogd, @linas?

File: CMakeLists.txt [hunk 1]
```diff
@@ -508,8 +508,8 @@ ADD_DEFINITIONS(-DDATADIR=\\"${DATADIR}\\")
 SET (MAN_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/man")
 
 # include custom opencog cmake macros
-INCLUDE("${COGUTIL_DATA_DIR}/cmake/OpenCogMacros.cmake")
-INCLUDE("${COGUTIL_DATA_DIR}/cmake/OpenCogFunctions.cmake")
+INCLUDE("${DATADIR}/cmake/OpenCogMacros.cmake")
```
Comment (vsbogd) [inline CMakeLists.txt]: ```COGUTIL_DATA_DIR``` and ```DATADIR``` can be different.
```COGUTIL_DATA_DIR``` is a place where ```cogutil``` module is installed.
```DATADIR``` in ```opencog``` is a place where opencog will be installed.
```OpenCogMacros.cmake``` is located in ```cogutil``` module so ```COGUTIL_DATA_DIR``` is correct to use here.

**Comment by linas** [comment]
This appears to be correct. `OpenCogMacros.cmake` accidentally got moved to cogutils; this restores the dependency to the atomspace.

**Comment by linas** [comment]
Per issue opencog/atomspace#1923

**Pull request merged**